docs/configuration.rst

```diff
@@ -164,6 +199,33 @@ Removes a variant set from a given data repository and dataset.
 
     $ ga4gh_repo remove-variantset path/to/repo.db aDataset aVariantSet
 
+++++++++++++++
+add-featureset
+++++++++++++++
+
+Adds a feature set to the repo created using `scripts/generate_gff3_db.py`.
```

Comment:
I'm not sure how helpful this actually is --- we don't distribute the ``generate_gff3_db.py`` script, so this is only going to cause confusion. We'll need to make this a first class CLI program, and document it separately. This needs to be done in a follow up issue, where we look at the requirements for importing featuresets properly and figure out the workflow for the administrator.